torchaudio/compliance/kaldi.py

```diff
@@ -512,3 +514,172 @@ def fbank(
         mel_energies = mel_energies - col_means
 
     return mel_energies
+
+
+def get_LR_indices_and_weights(orig_freq, new_freq, output_samples_in_unit, window_width,
+                               lowpass_cutoff, lowpass_filter_width):
+    """Based on LinearResample::SetIndexesAndWeights where it retrieves the weights for
```

Comment:
This also seems to be based on linear resampling. If so, the naming/parameters should make the assumption explicit, as suggested [here](https://github.com/pytorch/audio/pull/134/files#r302655143).